src/seml/start.py

```diff
@@ -1238,53 +1267,16 @@ def start_jupyter_job(
         exit(1)
 
     logging.info('Slurm job is running. Jupyter instance is starting up...')
-    log_file_contents = ''
+
     # Obtain list of hostnames to addresses
-    hosts = subprocess.run(
```

Comment:
I guess you removed that since the hostname is now available on our cluster, right?